dags/hivemind_etl_helpers/gdrive_ingestion_etl.py

```diff
@@ -0,0 +1,70 @@
+import logging
+
+from hivemind_etl_helpers.src.utils.credentials import load_redis_credentials
+from llama_index.core.ingestion import (
+    DocstoreStrategy,
+    IngestionCache,
+    IngestionPipeline,
+)
+from llama_index.core.node_parser import SemanticSplitterNodeParser
+from llama_index.storage.docstore.postgres import PostgresDocumentStore
+from llama_index.storage.kvstore.redis import RedisKVStore as RedisCache
+from llama_index.vector_stores.postgres import PGVectorStore
+from tc_hivemind_backend.db.credentials import load_postgres_credentials
+from tc_hivemind_backend.embeddings.cohere import CohereEmbedding
+
+
+class GoogleDriveIngestionPipeline:
+    def __init__(self, community_id: str):
+        self.community_id = community_id
+
+        # Embedding models
+        self.cohere_model = CohereEmbedding()
+        self.pg_creds = load_postgres_credentials()
+        self.redis_cred = load_redis_credentials()
+        self.table_name = "gdrive"
+        self.dbname = f"community_{community_id}"
+
+        self.db_config = {
```

Comment:
Please consider passing the values directly to line 52 rather than creating new `db_config` and passing db_config.

Reply:
okay